src/components/gallery/Picture.jsx

```diff
@@ -1,8 +1,22 @@
-const Picture = () => {
+"use client";
+import React, { useState } from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+const Picture = ({ text, link, image }) => {
+  const [isHovered, setIsHovered] = useState(false);
+
   return (
-    <div className="">
-      <p>Picture</p>
-    </div>
+    <Link href={link} target="_blank">
+      <div className="relative flex justify-center items-center">
+        {isHovered && <p className="absolute">{text}</p>}
+        <Image
```

Comment:
use tailwind hover for the image instead of using hooks.
https://tailwindcss.com/docs/hover-focus-and-other-states